### PR TITLE
Intake

### PR DESCRIPTION
**Pull request by Eshan41**
Made by Pradyun Pranshu Eshan Ahana

### CODE REVIEW

**Review by Jolteous — APPROVED** [review]
Good job, could use some updates, but for now let's merge this.

File: src/main/java/frc/robot/Commands/RunVelocity.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants.IntakeConstants;
+import frc.robot.Subsystems.IntakeSubsystem;
+
+/* You should consider using the more terse Command factories API instead https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands */
+public class RunVelocity extends Command {
+  /** Creates a new RunVelocity. */
+  private IntakeSubsystem intakeSubsystem;
+  private double reachVelocity;
+  PIDController pidController = new PIDController(IntakeConstants.kP, IntakeConstants.kI, IntakeConstants.kD);
```
Comment (MostOriginalIGN) [inline src/main/java/frc/robot/Commands/RunVelocity.java:17]: Should have been initalized in initalize instead but will not cause any immediate issues.

**Review by MostOriginalIGN — APPROVED** [review]
Looks great other than a small warning and some naming conventions.

**Comment by flashrun24** [comment]
You forgot a semicolon, 
image

**Review by Jolteous — CHANGES_REQUESTED** [review]
You're missing a semicolon. Check that the project builds before making a commit.

File: src/main/java/frc/robot/Commands/RunIntakeWithController.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+package frc.robot.Commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+/* You should consider using the more terse Command factories API instead https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands */
+public class RunIntakeWithController extends Command {
+  private final CommandXboxController joystick;
+  /** Creates a new RunIntakeWithController. */
+  public RunIntakeWithController( CommandXboxController joystick) {
```
Comment (MostOriginalIGN) [inline src/main/java/frc/robot/Commands/RunIntakeWithController.java:13]: Bad constructor. Requires intakeSubsystem but is not passed in.
```suggestion
  public RunIntakeWithController( CommandXboxController joystick, IntakeSubsystem intakeSubsystem) {
```

File: src/main/java/frc/robot/Commands/RunIntakeWithController.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    intakeSubsystem.coast();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    if ((joystick.getRightBumperPressed || joystick.getLeftBumperPressed()));
```
Comment (MostOriginalIGN) [inline src/main/java/frc/robot/Commands/RunIntakeWithController.java:28]: getRightBumperPressed needs to be called as a function
```suggestion
    if ((joystick.getRightBumperPressed() || joystick.getLeftBumperPressed()));
```

File: src/main/java/frc/robot/Commands/RunIntakeWithController.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      intakeSubsystem.setPower(0.9)
```
Comment (MostOriginalIGN) [inline src/main/java/frc/robot/Commands/RunIntakeWithController.java:29]: missing ;
```suggestion
      intakeSubsystem.setPower(0.9);
```

**Review by MostOriginalIGN — CHANGES_REQUESTED** [review]
Several breaking issues in this code. Please make sure you check your code for errors and verify that it builds.

**Comment by CryptoSOL123** [comment]
My team made these changes after the pull request. Do the changes show on the master branch after we push to it? I didn't have time to fix the errors that were made after the pull request since more stuff was added.

**Comment by MostOriginalIGN** [comment]
No, changes do not show on the master branch after you push to it. After a pull request is created, it captures the changes made up to that point, and additional changes will only show in the PR if they are pushed to the same branch associated with it. If your team made changes after the PR and pushed them, they should appear in the PR if they were committed to the correct branch; however, they won’t show in the master branch unless the PR has been merged into master and the branch is updated both locally and remotely. 

This PR did not need to be closed and you can make the necessary changes. Or create a new pull request when you are ready.

**Pull request closed without merging**